.github/workflows/pr-open.yml

```diff
@@ -72,7 +72,5 @@ jobs:
             -p PROMOTE=${{ github.repository }}/frontend:${{ github.event.number }}
             -p REACT_APP_USER_POOLS_ID=${{ vars.REACT_APP_USER_POOLS_ID }}
             -p REACT_APP_USER_POOLS_WEB_CLIENT_ID=${{ vars.REACT_APP_USER_POOLS_WEB_CLIENT_ID }}
-            -p REACT_APP_REDIRECT_SIGN_OUT="${{ vars.REACT_APP_REDIRECT_SIGN_OUT }}"
```

Comment:
Is `REACT_APP_REDIRECT_SIGN_OUT` not required anymore?

Reply:
It is for PROD, since that's the only place this value different from the default.